lib/public_key/src/public_key.erl

```diff
@@ -1228,6 +1232,41 @@ pkix_test_data(#{} = Chain) ->
 pkix_test_root_cert(Name, Opts) ->
     pubkey_cert:root_cert(Name, Opts).
 
+%%--------------------------------------------------------------------
+-spec ocsp_status(binary(), list(), undefined | binary()) ->
+    {ok, term()} | {error, term()}.
+%%
+%% Description: Verify the OCSP response to get the certificate status
+%%--------------------------------------------------------------------
+ocsp_status(OCSPResponseDer, ResponderCerts, Nonce) ->
+    catch pubkey_ocsp:verify_ocsp_response(
+        OCSPResponseDer, ResponderCerts, Nonce).
+
+%%--------------------------------------------------------------------
+-spec ocsp_extensions(undefined | binary()) -> list().
+%% Description: Get OCSP stapling extensions for request
+%%--------------------------------------------------------------------
+ocsp_extensions(Nonce) ->
+    [Extn || Extn <- [pubkey_ocsp:get_nonce_extn(Nonce),
+                      pubkey_ocsp:get_acceptable_response_types_extn()],
+             erlang:is_record(Extn, 'Extension')].
+
+%%--------------------------------------------------------------------
+-spec get_ocsp_responder_id(#'Certificate'{}) -> binary().
+%%
+%% Description: Get the OCSP responder ID der
+%%--------------------------------------------------------------------
+get_ocsp_responder_id(Cert) ->
```

Comment:
I think we can remove get from the function name. It is good if all API functions start with ocsp and get does not add much value, and none of the other  API functions have "get" in their names.  See for example public_key:pkix_issuer_id/2

Reply:
> I think we can remove get from the function name. It is good if all API functions start with ocsp and get does not add much value, and none of the other API functions have "get" in their names. See for example public_key:pkix_issuer_id/2

done